### ISSUE TITLE
[ENDEAVOUR] Decks by the Numbers

### ISSUE THREAD

**Issue by KillianKirilenko** [Citadel-Station-13/Citadel-Station-13-RP#1500]
I'm just going to look through this in DreamMaker and throw out thoughts as I go!

**DECK ONE**
- As some have mentioned, the Viro Shuttle walls need to be a different type.
- Both port and starboard fore sections of this deck have a *lot* of dummy maintenance that could be repurposed into useful space for Medical and Science alike.
- Science has this really awkward 3-wide corridor running through most of it. Cut that down to 2-wide like most of Medical to get some space back.
- Ditto, axe the shuttle corridor down to 2-wide. The airlock will be able to cycle faster with less tiles to work (not that it should ever need to cycle, strictly speaking).
- RD's office window setup is goofy as fuck, as FFS mentioned.
- Someone went absolutely ham placing trash piles around tcomms (aft port maint). A few of these could probably stand to be axed or moved into other areas.
- Speaking of tcomms, as others have mentioned it's not very well protected and may need some tweaking.

**DECK TWO**
- I'm not really qualified to speak on Engineering or Atmos, so I won't.
- Remember what I said about dummy maint space on Deck 1? The Break Room and Patient Rooms for Medical can be dropped down if that space is rearranged, instead of being this weird little isolated space on a lonely side-staircase.
- The Command Bar seems like a cute idea on paper but as others have suggested elsewhere I think it's a bad idea long term. Command should by all means have a quiet breakroom/conference room, but a whole bar? Nah.
- Instead, consider replacing it with an armored vault. This map lacks one, and it's a possible target for antags and events down the line - heists, etc.

**DECK THREE**
- Oof, that double-window, non-electrified bridge "meeting room". If need be just make the whole thing out of a custom polarizable reinforced window.
- The space outside the meeting room is considered the "Bridge Hallway", whereas the unpowered 'bubble' is considered the Bridge itself. Flip these!
- Lots of dummy maint up here too.
- As others have noted, the NT Breacher needs to be in a more secure location if it's to continue existing.

**DECK FOUR**
- I've said it a dozen times and I'll say it a dozen more: the Explo shuttle is a turgid abomination. I intend to experiment with alternate designs with new layouts when I have some free time/energy.
- Aside from that Exploration ain't too bad! It's well-equipped but not overly so. I think my only real 'issue' is that it seems kind of weird that exploration meeting looks out immediately over a solar array.
- Tether has the PF's office between prep and meeting so you can easily keep an eye on both of them. I'd suggest considering doing this here too. Make Prep where Meeting is, PF's Office over Prep, and Meeting over PF's Office. This also means the Meeting Room can look straight out into space without the solars in the way.
- There's an absolutely egregious amount of dummy maint space on portside fore of cargo.
- The PF's office doesn't need a GPS on top of the locker; it's a default locker item now.

**DECK FIVE**
- Like Med/Sci, Sec has this awful three-tile-wide corridor. Cut it down and reclaim some space!
- The Construction Site is a thing, I guess? Give it some windows to the outside at the very least.
- Fucking christ there's a three-deep block of rwalls fore of Sec's firing range. What are they being given that can potentially punch through that much? Literally nothing. It's a practice range, they should be using practice lasers and blank/practice ballistic rounds. Cut this down to doublewide max.
- I realize the CE and RD get their own hardsuits, but giving the HoS their own is asking for someone to wear it ready-but-undeployed 24/7, especially if antag/event rounds become more common. I suggest replacing this with the proto-suit ASAP. That said, I do like the idea that it has a few extra modules *just* for the HoS. Maybe keep those in one of the lockers.

**GENERAL**
- Excessive amounts of triple-wide corridors. Many of these could easily be knocked down to double-wide to reclaim space without significantly limiting mobility or accessibility. The triple-wide central corridor makes sense, but elsewhere not so much.
- The CE's office is the only place with a fireaxe cabinet. Tether has one in the EMT Bay, one in Atmos, and one on the Bridge.
- The Elevator is much Fatter than it needs to be and could probably be cut down to more reasonable proportions as a double-elevator. I suggest two 3x2s with the inner divider and outer walls made of glass. The free 2x5 aft of the elevator on each deck can be repurposed into corridor, maintenance, or etc. space as desired (alternately it could be some kind of special/cargo elevator that skips Deck 2).
- The use of metal foam as literal filler is a cute but inconsistent touch. I'd honestly suggest filling the outermost 'ring' of maint tunnels with it entirely for consistency. Then there won't seem like there's so much wasted maint space, I think.

*In Summary*
The map seems to suffer from a lot of excessive maint-padding and wasted space, and the maint space that isn't pure padding is very bland - there are no 'scene rooms' like Tether's secret fight club/hidden cybernetics lab/etc. and very few quiet repurposeable areas like the Abandoned Locker Room or Surf2 Bar Maint, which some people might appreciate. Be sure to use fake walls to make "inaccessible" areas.

Basically, have some defined 'rooms' within Maint that can easily be repurposed.

That's all for now! I might add more ingame experience based feedback on here later if I can actually be around for full playtests.

**Comment by KillianKirilenko** [comment]
Deck Four: Exp. Shuttle Rework Proposal;

![image](https://user-images.githubusercontent.com/49700375/63947651-36bc4700-ca6f-11e9-9df7-4b8c6097b3a2.png)

Inverted 'Streamlined' design. The port-side serves as a large three-door airlock with plenty of room for people to move around and space for cargo/loot/etc., whilst the starboard-side has cargo and the airlock tanks. Has a door bolt control in the cockpit.

**Comment by Andromeda-K22** [comment]
- **Deck One**
Viro shuttle has new walls. I'm not sure how the dummy space can be used. A re-arrange of stuff in Science is probably in the cards in the future, but until then cutting it down to 2-tiles would be a fairly large amount of work. Research shuttle airlock was trimmed down. RD's office was un-windowed. Number of trash piles was cut in half near tcomms and tcomms security was buffed up with turrets and associated controls.

- **Deck Two**
I have resolved the engineering/atmos issues that I am currently aware of. As said, not entirely sure how to utilize the dummy space. Ideas are welcome. Command bar is likely going to be removed some time soon and replaced with something else, likely a vault and or a self-destruct device, pending a command remap.

- **Deck Three**
The meeting room is going to be suddenly more ... *shocking!* Which means i've electrified the grills. Polarized windows were already present, but it's likely the maint-tunnel bit will be removed in favor of shifting things around soon-ish. NT Breacher removed. Nobody uses it anyway. 

- **Deck Four**
Suggestions welcome for the shuttle, but until I have a valid design worked out, Fatso shall remain. This deck is in general dummy thicc pending remaps and such, maint is dummy space to run things through and shift things around into eventually. Will definitely shift the PF's office around soon enough.

- **Deck Five**
Construction site is gonna go byebye with the Command remap. Dummy thicc bulletsponge removed. Protosuit is honestly kinda useless, and I figured the hardsuit would be a change of pace - it's an administration issue in my eyes, and if it proves to be a problem, it *will* be replaced with the protosuit.

- **General**
Triplewides will be reduced where needed. More fire-fighting axes will be distributed to the madmen. Elevator is pending changes soon-ish, when I can figure out where to put more of them and how to deal with the removal of the singular monolithic elevator. ~~teshari removal pit~~

I don't like the idea of filling the inner ring of maint with metal foam. The space is there for a reason, frankly. I think it's a waste of effort to just fill it up when it serves a decent purpose - i.e, if a space is blocked, maint is a way around the issue.

**Comment by KillianKirilenko** [comment]
The Protosuit is honestly one of the better well-rounded suits, though? It's basically second only to the Director's Voidsuit in all-round protection (a couple of specialist suits are better in key areas but worse for general threats). The Hazard Hardsuits are only *marginally* superior in terms of defenses, though I'll admit they're vastly more useful in terms of sheer utility.

It's in a bit of an odd spot though.

No thoughts on the shuttle proposal image? I actually have map files with a fully working version that I've tested out pretty thoroughly. The airlock controls are a *little* touchy compared to Tether's (the doors can very easily be locked open, exposing the passenger compartment to space) but work fine once planetside.

Only issue with it is that the 5x3 airlock takes a month and a year to cycle. Probably easiest to just squish airlock+cargo back a bit and enlarge the passenger bay.

**Comment by Andromeda-K22** [comment]
My main issue is, well, it's *small*. Smaller than Tether's shuttle, even. The hangar on Endeavour has a lot more space than that.

**Comment by KillianKirilenko** [comment]
It's thinner, yes, but remember it's longer. Overall, the streamined shuttle actually has more free floor space. Tether proves that bigger isn't always better, you know?

I have some plans to make it a little sleeker-looking (right now it's awfully bland) and rejigger the cargo/airlock space a touch. Will hash that out and post a preview soon.

**Comment by KillianKirilenko** [comment]
![image](https://user-images.githubusercontent.com/49700375/64129804-58426900-cdb6-11e9-8185-5718d9a5770d.png)

Sleeker, and now with even more space for loot. On the other hand I broke the airlock. I hate this thing sometimes.

**Comment by Andromeda-K22** [comment]
Design looks nice. That massive airlock needs to go, though. Whew.

**Comment by KillianKirilenko** [comment]
See, I have a crazy solution to that: instead of having the airlock on the shuttle itself, make it part of any exploration map that requires it,

Seems like a win-win to me. No more fiddly airlock, which can be used for a mini-medbay or something.

**Comment by KillianKirilenko** [comment]
![image](https://user-images.githubusercontent.com/49700375/64278477-bf8e2380-cf44-11e9-8751-4af885a9552b.png)
Here's an airlock-less design prototype, with the main body brought in a tiny bit to lend it a sleeker overall profile.
